### ISSUE TITLE
ssl.google-analytics.com in the log file

### ISSUE THREAD

**Issue by buhtz** [tinyMediaManager/tinyMediaManager#195]
2.9.4 snapshot from git on Debian GNU/Linux unstable

I found this line in the log
`
2017-08-22 21:55:37,225 DEBUG [trackEventThread] org.tinymediamanager.scraper.http.Url:241 - getting https://ssl.google-analytics.com/collect?v=1&tid=UA-35564534-5&cid=0acb1948-a4b6-4e7c-9612-8cc246cbe7e4&an=tinyMediaManager&av=2.9.4-SNAPSHOT&t=event&ec=ping-pong&ea=ping-pong&aip=1&je=1&ul=de-DE&vp=0x0&cd1=Linux&cd2=amd64&cd3=1.8&cd4=2.9.4-SNAPSHOT&cd5=0&z=1503431737225&sr=1920x1200
`

What is happening here? What has TMM has to do with Google?

There is just a 1x1 white pixel behind it.

**Comment by ProIcons** [comment]
Analytics. Exactly what you are seeing.

**Comment by buhtz** [comment]
Can you explain it a bit more detailed. What does tmm analysis here? It is not webbrowser?
Is there a EULA or something like this informing the user about that a third party is collecting data?

**Comment by myron0815** [comment]
You can turn this off in settings, first screen.

It's just some basic stats we collect, like tmm/os/java versions, screen resolution and regional things...
Helps us focusing on the most-used systems

**Comment by buhtz** [comment]
I was not aware that this could be done with Google technic. There should be a hint to the user that Google is used for that. This would be fair.

**Comment by mlaggner** [comment]
as stated myron above there is only anonymized data which will be sent to google (you find everything in which is transferred in call from the log).
should be irrelevant where this is collected

**Comment by mlaggner** [comment]
after having a talk with myron we will add a disclaimer with all relevant infos about tmm in the wizards first page..
will be in v3

**Comment by buhtz** [comment]
Thanks for getting into this and handling this.
But just to clearify: The data is not "anonymized"! It is "pseudomized" what is a big difference. And you know that Google connets data from different sources to create an identiy out of it. And it is fact that Google root all it's data to NSA & Co.
I would recommend to overthink this service and value the benefits against the costs.

**Comment by buhtz** [comment]
The problem occurse in the `v3` branch also.
There I see no hint about google in the settings dialog.

**Comment by ProIcons** [comment]
Is it hard for you to understand that is under development and not public release? We already stated that over 5 times.

**Comment by mlaggner** [comment]
the not is in the disclaimer in the wizard - if you don't read it, don't complain about it

**Comment by buhtz** [comment]
@ProIcons I understand. I have no problem with the unstable status of `v3`. But maybe I have a problem to understand your project-intern workflow. In most projecst unstable versions, snapshots, pre-releases or just git-repos are for testing. Of course there are bugs in them - because they are not stable. So it is a chance for the developers to let others test this and finding bugs. Maybe you TMM-devs see this other? __This question is serious!_ You don't want someone using `v3` (or other unstable version of TMM) and report bugs and ideads?

**Comment by buhtz** [comment]
@mlaggner I see. What do you think about adding the telemetry-setting in the wizard window for TMM's first start? In that case the user could decide to use this or not __before__ the first contact to Google.

**Comment by ProIcons** [comment]
@buhtz as far of your response to me i gotta say, that user feedback is very important to us. In a previous comment Project's founder @mlaggner stated that this will br added on v3. Beside the fact that a commit regarding the very same issue was pushed shortly after his response, you need to understand something about tmm. Manuel, and the other Contributors including me, are developing this software on our spare time. We have paying jobs which consumes a lot of our time and energy so while we move on, after some time our energy and spare time are getting decreased in exponentially. While that doesn't mean that we will cease the development of tmm, it means that the implementation of some things might take a while. So don't expect from the very moment you got the response that a feature will be implemented to br implemented in the same second, minute, hour, day, week or even month. 

As far of your response to Manuel, while i haven't compiled the new v3 branch after Manuel' s push in reviewed the code he pushed and i'm pretty sure  he disabled the Google Analytics by default and added an option to enable them on the Wizard.

**Comment by buhtz** [comment]
> So don't expect from the very moment you got the response that a feature will be implemented

I don't do. I just interpreted @mlaggner comment and the comit https://github.com/tinyMediaManager/tinyMediaManager/commit/522a87c88302d1d631039ee57533bf2baffef609 that it was still done. Then I tried it in `v3` and didn't saw it there. I assumed that the commit was done in `master` not in `v3`. And I currently checked my local clone of your git. The commit is in here.

Don't feel pushed or something like that. I am a dev, too and know how ressource management goes in such projects. This is just a BugTrackere here. So keep calm and don't become to emotional. It is just for documentation of problems and wishes. I think your reactions are caused by missinterpretation of my.

And I don't understand the things you wrote about compilation. Maybe I miss some infos about Java. But I use the git repo and build this myself. So IMO it doesn't matter if you compiled something or not. Or not? ;)
